### PR TITLE
Update mkl_verbose return value check due to API change in mkl

### PR DESCRIPTION
**Pull request by jingxu10**
As title.
Originally `mkl_verbose()` function returned `0` and `1`, indicating failure and success respectively. However, the version that PyTorch uses now changed the output of `mkl_verbose()` to reflect its input level. Thus, the check logic needs to be changed to compare output of the `mkl_verbose()` function with -1.
https://www.intel.com/content/www/us/en/develop/documentation/onemkl-developer-reference-c/top/support-functions/miscellaneous/mkl-verbose.html

cc @frank-wei @jgong5 @mingfeima @XiaobingSuper @sanchitintel @ashokei

### CODE REVIEW

**Review by jgong5 — APPROVED** [review]

**Comment by jingxu10** [comment]
Hi @malfet , would you help to review this PR? Thank you.

**Comment by jingxu10** [comment]
Hi @malfet , would you help to review this PR? Thank you.

File: aten/src/ATen/native/verbose_wrapper.cpp [hunk 1]
```diff
@@ -14,7 +14,8 @@ namespace verbose {
 
 TORCH_API int _mkl_set_verbose(int enable) {
```
Comment (malfet) [inline aten/src/ATen/native/verbose_wrapper.cpp:15]: Do you mind changing return type to bool to indicate the intent here?
```suggestion
TORCH_API bool _mkl_set_verbose(int enable) {
```

**Review by malfet — APPROVED** [review]
Looks good to me, but please consider changing output to `bool` to make the semantic more explicit.

File: aten/src/ATen/native/verbose_wrapper.cpp (hunk 1 above)
Comment (jingxu10) [reply aten/src/ATen/native/verbose_wrapper.cpp:15]: Yeah, sure. But will need to update other files as well in this invocation chain. I'll do it later.

Comment (malfet) [reply aten/src/ATen/native/verbose_wrapper.cpp:15]: At the very least, add a comment, that it expects to return 0 on failure and 1 on success

Comment (jingxu10) [reply aten/src/ATen/native/verbose_wrapper.cpp:15]: OK. I'll add the comment.

**Comment by kit1980** [comment]
I'll reland if the revert will not repair the test (now that I looked closer, the PR is probably unrelated).

**Pull request closed without merging**